src/tailwind.css

```diff
@@ -55,6 +55,11 @@
   --color-text-primary: var(--grey-700);
   --color-text-secondary: var(--grey-400);
   --color-text-tertiary: var(--yellow-800);
+
+  /* SPECIFIC UI ELEMENTS */
+  --color-background-button: var(--color-action-primary);
```

Comment:
Wondering if these should be prefixed `--ra-` to avoid them being overridden accidentally by the host page if they also use this naming convention? That’s what I did in the original IRA embed CSS but it was defensive and not based on any specific evidence that there was clobbering in the wild. Open to trying this and then revising if needed.

Reply:
The shadow DOM prevents definitions in the host page from reaching here, so I don't think that's a concern.

There's another possible justification for prefixing, though, which is to distinguish supported and non-supported customization points (both for ourselves as embed developers, and for embed clients). We can document "you can override any variable that starts with `--ra-`; override anything else at your own risk".